app/data/models/associates.py

```diff
@@ -15,7 +15,7 @@ class Associate(Base):
     zip_code = sa.Column(sa.String(45), nullable=False)
     city = sa.Column(sa.String(100), nullable=False)
     country = sa.Column(sa.String(100), nullable=False)
-    associates_role = sa.Column(sa.String(100), nullable=False)
```

Comment:
Any reason role/category is changed to sqlalchemy relationship instead of value in DB? What i can see with this change is that we dont have any way to specify if the associate is a manufacturer or supplier.

Reply:
fixed relations